config/webpack/development.js

```diff
@@ -1,5 +1,16 @@
-process.env.NODE_ENV = process.env.NODE_ENV || 'development'
+const dotenv = require('dotenv');
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'require' is not defined  no-undef